systemtests/src/main/java/io/enmasse/systemtest/amqp/AmqpClientFactory.java

```diff
@@ -106,10 +109,16 @@ public AmqpClient createClient(TerminusFactory terminusFactory, ProtonQoS qos, A
             clientOptions.setSniServerName(messagingEndpoint.getHost());
         }
         log.info("External endpoint: " + clientEndpoint + ", internal: " + messagingEndpoint);
+        probeEndpoint(messagingEndpoint);
 
         return createClient(terminusFactory, clientEndpoint, clientOptions, qos);
     }
 
+    private void probeEndpoint(Endpoint messagingEndpoint) throws IOException {
+        Socket socket = new Socket(messagingEndpoint.getHost(), messagingEndpoint.getPort());
```

Comment:
I don't think the timeout behaviour of Socket(messagingEndpoint.getHost(), messagingEndpoint.getPort()) is too clearly specified.  I'd suggest using the no-arg form and passing an explicit timeout.